src/decisionengine/framework/modules/EmptySource.py

```diff
@@ -0,0 +1,32 @@
+"""
+This dummy source takes the name of a source datablock
+from config file as parameter "datablock_type" and produces
+an empty pandas DataFrame as a datablock with that name
+"""
+import pandas as pd
+
+from decisionengine.framework.modules import Source
+from decisionengine.framework.modules.Source import Parameter
+
+
+@Source.supports_config(
+    Parameter("datablock_type", default=""),
```

Comment:
A datablock is composed of data products--I would therefore call this `data_product_name` or something like that.  I wouldn't put the string "type" in it as we don't want to confuse the name with the Python type.

Reply:
OK. Made the change.